src/cart/SConscript

```diff
@@ -22,6 +22,21 @@ SRC = ['crt_bulk.c', 'crt_context.c', 'crt_corpc.c',
        'crt_swim.c', 'crt_tree.c', 'crt_tree_flat.c', 'crt_tree_kary.c',
        'crt_tree_knomial.c', 'crt_hlc.c', 'crt_hlct.c']
 
+LIB_SRC = ['crt_utils.c']
+
+##############################
+# Create cart utility library
+##############################
+def build_utility_library():
+    """Build cart utility library"""
+
+    denv = env.Clone()
+    prereqs.require(denv, 'mercury', headers_only=True)
+    cart_utils_lib  = daos_build.library(denv, 'cart_utils', LIB_SRC)
```

Comment:
This still requires you to install the library.  I want to remove that altogether.   Just do

cart_utils_objs = denv.SharedObject(LIB_SRC)

instead and you can remove all the spec changes.

Reply:
Make it shared objects.